app/helpers/handle-url.js

```diff
@@ -1,12 +0,0 @@
-import { helper } from '@ember/component/helper';
-
-export function handleUrl(url) {
-  var doc = document.createElement('a');
-  doc.href = url;
-  if(doc.hostname === 'pix.fr') {
-    return doc.pathname;
-  }
-  return url;
```

Comment:
plus de `return url` ?

Reply:
Je supprime cet helper car il n'était utilisé que par les menus contribuables. Seulement maintenant les menus utilisent le component `handle-link` qui lui utilise le helper `as-route` seulement quand il s'agit d'une adresse Pix.

Reply:
j'ai pas tout compris. Ce que je vois c'est que le code ici a été copié et collé dans un component, sauf le `return url`, du coup je me demande

Reply:
Alors avant on utilisais le helper `handle-url` afin de retourner l'url complète si c'était pas une adresse `https://pix.fr` et sinon juste le pathname permettant de garder la navigation dans les reviews apps par exemple. Maintenant, le component gère si l'adresse est une adresse Pix ou non, permettant si oui de faire un `link-to`.